Increase amount of text in character count threshold example (#2975)
alphagov/govuk-design-system: src/components/character-count/index.md

When using a character count, try to set the limit higher than most users will need. Find out what this is by doing user research and data analysis.

#### If the limit is far higher than users are likely to reach

> You can choose to display a character count message only after the user has entered a certain number of characters. This is useful when a character limit is needed, but users are unlikely to reach the limit.

Reviewer: Technically, it's not about the user entering characters, it's down to a threshold being passed. For example, if the text area is pre-filled, that'll activate the threshold condition as well.
```suggestion
You can choose to display a character count message only after the length of the text has surpassed a certain 'threshold' number of characters. This is useful when a character limit is needed due to the technical limitations of the service, but users are unlikely to reach the limit.
```

Author: This might be a teensy bit misleading now as it implies that the threshold is set based on the number of characters, when it's actually a percentage of the total limit.

Reviewer: How about just removing the word 'number' entirely then?
> You can choose to display a character count message only after the length of the text has surpassed a certain 'threshold' of characters. This is useful when a character limit is needed due to the technical limitations of the service, but users are unlikely to reach the limit.

Reviewer 2: We might as well go all the way and explain what the threshold is:

```suggestion
You can choose to display a character count message only after the length of text has passed a certain 'threshold', set as a percentage of the total character limit. This is useful when a character limit is needed due to the technical limitations of the service, but users are unlikely to reach that limit.```

Reviewer 2: This sentence is meant to be a bit of preamble before explaining explaining the threshold in the following sentence.

When using a character count, try to set the limit higher than most users will need. Find out what this is by doing user research and data analysis.

#### If the limit is far higher than users are likely to reach

> You can choose to display a character count message if the length of text within the textarea more than a certain 'threshold' of characters. This is useful when a character limit is needed due to the technical limitations of the service, but users are unlikely to reach the limit.
> 
> To do this, set the threshold in the component markup as a percentage. For example, `data-threshold="75"` will show the count message only when users have entered 75% of the limit.

Reviewer 2: ```suggestion
You can choose to display a character count message when the length of text within the textarea passes a certain 'threshold' of characters. This is useful when a character limit is needed due to the technical limitations of the service, but users are unlikely to reach that limit.

To do this, set the threshold in the component markup as a percentage. For example, `data-threshold="75"` will show the count message only when the user has entered a length of text that's 75% of the limit or more.
```
